oak-search-elastic/src/test/java/org/apache/jackrabbit/oak/plugins/index/elastic/ElasticConnectionRule.java

```diff
@@ -52,28 +53,19 @@
 public class ElasticConnectionRule extends ExternalResource {
 
     private static final Logger LOG = LoggerFactory.getLogger(ElasticConnectionRule.class);
-    private ElasticConnection elasticConnection;
-    private final String elasticConnectionString;
+
     private static final String INDEX_PREFIX = "elastic_test";
     private static final String PLUGIN_DIGEST = "060117b4150c87274d9cff0925ec16e714f28a40906a53a2cd2a23322bbb3189";
     private static boolean useDocker = false;
 
+    private final String elasticConnectionString;
+
     public ElasticConnectionRule(String elasticConnectionString) {
         this.elasticConnectionString = elasticConnectionString;
     }
 
     public ElasticsearchContainer elastic;
 
-    /*
-    Executed once in the test class' execution lifecycle, after the execution of apply()
```

Comment:
I don't recall exactly, but I think the purpose of getting the docker connection in the before method here was to make sure that docker container is run only once during the whole test class' execution - otherwise it would probably run and get destroyed after every test - leading to increased test execution time.

Reply:
I thought the same. But this was actually not needed. I confirm all tests in a class run against the same docker container.